### DISCUSSION TITLE
Plataformas en la Nube para subir imágenes

### DISCUSSION THREAD

**Discussion by pandafrontdeveloper** [community/community#81492] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Siempre he manejado mis imágenes en el lado del Backend, pero ahora quisiera manejarlas en la nube. ¿Qué plataformas me recomiendan?

**Comment by ignaciodiazdev** [comment, marked as answer]
Me parece excelente idea que empieces a investigar y considerar la gestión de imágenes en la nube, ya que en entornos profesionales, el manejo eficiente de recursos como las imágenes es crucial, y las plataformas en la nube ofrecen soluciones robustas para ello. Con esta práctica puedes optimizar la entrega de contenido multimedia, mejorar la experiencia del usuario y en muchos casos, reducir costos asociados con el almacenamiento y la transferencia de datos.

### Aquí te dejo el nombre de algunas plataformas que me han sido de ayuda:

1. **Cloudinary**
2. **DigitalOcean**
3. **Azure Blob Storage**
4. **Cloud Storage for Firebase**

> Cada plataforma tiene sus propias fortalezas, por lo que deberías evaluar cuál se adapta mejor a los requisitos del proyecto que realices.
